build.gradle

```diff
@@ -39,20 +36,19 @@ deploy {
         }
     }
 }
- 
+
 repositories {
     flatDir {
         dirs 'libs'
     }
 }
 
 // Set this to true to enable desktop support.
-def includeDesktopSupport = true
+def includeDesktopSupport = false
```

Comment:
Seems like the above would disable simulation which is proabably not what we want.

Reply:
I didn't notice any problems while testing it, however, I have changed it back to true